### ISSUE TITLE
App crashed immediately after launch

### ISSUE THREAD

**Issue by mehocode** [avocode/avocode#1062]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.9.1
Source UID: 67766
```
Error: Downloader is disposed, cannot request a file download
    at e.downloadFiles (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:20:7906)
    at t.downloadFiles (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:13:15334)
    at e.exports.e._requestImages (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:14:533)
    at e.exports.e.downloadImages (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:13:31153)
    at e._downloadRevisionImages (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:11:21744)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:11:19903
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:13:5085
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:13:14316
    at file:///Applications/Avocode.app/Cont
```

**Comment by jakubzitny** [comment]
Hi @mehocode, thanks for submitting the issue. This is a bug in Avocode, we already know about it and we're working on a fix. Hopefully, it'll be available in one of the next releases. We'll keep you updated.

If you have any more questions or troubles regarding Avocode, let us know 😉

**Comment by connor-bean** [comment]
We just released 2.11 so I'm closing this out. Please let us know if this problem persists.

You can download the update at https://avocode.com/news.html

Thanks for your patience on this.

Connor, Avocode Support
